R/plot.shapes.R

```diff
@@ -505,14 +519,27 @@ add_shape <- function(shape, clip=shape_noclip,
   if (length(vertex.frame.color) != 1 && !is.null(v)) {
     vertex.frame.color <- vertex.frame.color[v]
   }
+  vertex.frame.width <- params("vertex", "frame.width")
+  if (length(vertex.frame.width) != 1 && !is.null(v)) {
+    vertex.frame.width <- vertex.frame.width[v]
+  }
   vertex.size        <- 1/200 * params("vertex", "size")
   if (length(vertex.size) != 1 && !is.null(v)) {
     vertex.size <- vertex.size[v]
   }
   vertex.size <- rep(vertex.size, length=nrow(coords))
   
-  symbols(x=coords[,1], y=coords[,2], bg=vertex.color, fg=vertex.frame.color,
-          squares=2*vertex.size, add=TRUE, inches=FALSE)
+  if (length(vertex.frame.width) ==1) {
+    symbols(x=coords[,1], y=coords[,2], bg=vertex.color, fg=vertex.frame.color,
+          squares=2*vertex.size, lwd=vertex.frame.width, add=TRUE, inches=FALSE)
+  } else {
+    mapply(coords[,1], coords[,2], vertex.color, vertex.frame.color,
+           vertex.size, vertex.frame.width,
+           FUN=function(x, y, bg, fg, size, lwd) {
+               symbols(x=x, y=y, bg=bg, fg=fg, lwd=lwd,
+                       square=2*size, add=TRUE, inches=FALSE)
```

Comment:
I think this should be `squares` here, not `square`. (The CI warns about this in a `NOTE`).

Reply:
just saw this, sorry bout that ;)